frontend/src/components/layout/Header/HeaderStyled.tsx

```diff
@@ -1,6 +1,10 @@
-@import '../../../fonts.scss';
+import { Navbar } from 'react-bootstrap';
+import styled from 'styled-components';
 
-.App-header.navbar {
+/**
+ * Styled header component.
```

Comment:
same thing, since this is a visual component we can describe:
1) intended use
2) what is displayed.